bin/tessel-2.js

```diff
@@ -7,6 +7,7 @@ var key = require('../lib/key');
 var init = require('../lib/init');
 var logs = require('../lib/logs');
 
+
```

Comment:
This is super nit-picky but before requesting reviews on PRs, can you remove whitespace changes?